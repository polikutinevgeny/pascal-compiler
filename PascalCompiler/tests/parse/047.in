var
   a, b, i, n: integer;
   function foo(var a, b: integer): integer;
   var
      i: integer;
   begin
      for i := 0 to n do
      begin
         result := result*20;
      end;
   end;

begin
   repeat
      n := 30;
      n := n + foo(a, b);
      a := n - 20;
   until a > 0;
end.