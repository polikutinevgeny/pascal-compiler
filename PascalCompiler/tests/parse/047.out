Illegal identifier at 9:10
